src/verso/Verso/Doc/Concrete.lean

```diff
@@ -185,151 +121,121 @@ def versoBlockCommandFn (genre : Term) (title : String) : ParserFn := fun c s =>
     else
       s.mkNode ``addBlockCmd iniSz
 
+/-! The elaboration of zero-or-more `addBlockCmd` calls, followed by th elaboration of
+`addLastBlockCmd`, are connected by state stored in environment extensions. -/
+
 initialize docStateExt : EnvExtension DocElabM.State ← registerEnvExtension (pure {})
 initialize partStateExt : EnvExtension (Option PartElabM.State) ← registerEnvExtension (pure none)
 initialize originalCatParserExt : EnvExtension CategoryParserFn ← registerEnvExtension (pure <| fun _ => whitespace)
```

Comment:
```suggestion
/--
The original parser for the `command` category, which is restored while elaborating a Verso block so that
nested Lean code has the correct syntax.
-/
initialize originalCatParserExt : EnvExtension CategoryParserFn ← registerEnvExtension (pure <| fun _ => whitespace)
```